assets/themes/campersites/campersites.json

```diff
@@ -0,0 +1,718 @@
+          },
+          "question": {
+            "en": "Would you like to add a general description of this place? (please keep it objective; opinions go into the reviews)"
+          },
+          "freeform": {
+            "key": "description",
+            "type": "text"
+          }
+        }
+      ],
+      "hideUnderlayingFeaturesMinPercentage": 0,
+      "icon": {
+        "render": "./assets/themes/campersites/caravan.svg",
+        "mappings": [
+          {
+            "if": {
+              "and": [
+                "fee=no"
+              ]
+            },
+            "then": {
+              "en": "./assets/themes/campersites/caravan_green.svg"
+            }
+          }
+        ]
+      },
+      "width": {
+        "render": "8"
+      },
+      "iconSize": {
+        "render": "40,40,center",
+        "mappings": [
+          {
+            "if": {
+              "and": [
+                "fee=yes"
+              ]
+            },
+            "then": {
+              "en": "This place charges a fee"
+            }
+          },
+          {
+            "if": {
+              "and": [
+                "fee=no"
+              ]
+            },
+            "then": {
+              "en": "You can use this place for free"
+            }
+          }
+        ]
+      },
+      "color": {
+        "render": "#00f",
+        "condition": {
+          "and": []
+        },
+        "mappings": [
+          {
+            "if": {
+              "and": [
+                "fee=no"
+              ]
+            },
+            "then": {
+              "en": "for free"
+            }
+          }
+        ]
+      },
+      "presets": [
+        {
+          "tags": [
+            "tourism=caravan_site"
+          ],
+          "title": {
+            "en": "camper site"
+          },
+          "description": {
+            "en": "Add a new official camper site. Please zoom in sufficiently for exact placement. Don't copy info from copyrighted content. Your data will be added straight to the global OpenStreetMap database!"
+          }
+        }
+      ],
+      "wayHandling": 2
+    },
+    {
+      "id": "dumpstations",
+      "name": {
+        "en": "Sanitary dump stations"
+      },
+      "minzoom": 12,
+      "overpassTags": {
+        "or": [
+          "sanitary_dump_station=yes",
+          "amenity=sanitary_dump_station"
+        ]
+      },
+      "title": {
+        "render": {
+          "en": "Dump station {name}"
+        },
+        "mappings": [
+          {
+            "if": {
+              "and": [
+                "name="
+              ]
+            },
+            "then": {
+              "en": "Dump station"
+            }
+          }
+        ],
+        "condition": {
+          "and": []
+        }
+      },
+      "description": {
+        "en": "Sanitary dump stations"
+      },
+      "tagRenderings": [
+        "images",
+        {
+          "render": {
+            "en": "fee"
```

Comment:
Same as above: rendering should use at least one {key}

Reply:
done